package.json

```diff
@@ -20,13 +20,13 @@
   "homepage": "https://github.com/matthew-andrews/isomorphic-fetch/issues",
   "dependencies": {
     "node-fetch": "^1.0.1",
-    "whatwg-fetch": ">=0.10.0"
+    "whatwg-fetch": "niftylettuce/fetch"
```

Comment:
for various reasons I'd prefer to keep this on the main version — not on someone's fork.  Why is this needed?